src/calculi/mod.rs

```diff
@@ -0,0 +1,121 @@
+mod ert;
```

Comment:
Please add a module comment in this file outlining what the module does (introducing the calculus annotations and that they check which proof rules can be used where).